config/sites/sitenow.uiowa.edu/config_split.patch.uiowa_auth.settings.yml

```diff
@@ -1,15 +1,9 @@
 adding:
   role_mappings:
-    - 'administrator|urn:oid:2.5.4.31|CN=ITS-WebServices,OU=Groups,OU=ITS,DC=iowa,DC=uiowa,DC=edu'
-    - 'administrator|urn:oid:2.5.4.31|CN=ITS-Students-WebServices,OU=Groups,OU=ITS,DC=iowa,DC=uiowa,DC=edu'
     - 'requester|urn:oid:1.3.6.1.4.1.5923.1.1.1.5|student'
     - 'requester|urn:oid:1.3.6.1.4.1.5923.1.1.1.5|faculty'
     - 'requester|urn:oid:1.3.6.1.4.1.5923.1.1.1.5|staff'
     - 'requester|urn:oid:1.3.6.1.4.1.5923.1.1.1.5|affiliate'
     - 'requester|urn:oid:1.3.6.1.4.1.5923.1.1.1.5|member'
   all_valid_hawkids_can_authenticate: true
```

Comment:
I've no idea what this is. Its in main but I don't think it should be. Going to remove...

Reply:
i plan to do the same for brand